pyproject.toml

```diff
@@ -62,7 +62,7 @@ dependencies = [
   "sparse",
   "toolz",
   "xarray >=2023.11.0, !=2024.6.0, <2024.10.0", # FIXME: 2024.10.0 breaks rechunker with zarr
-  "xclim >=0.53.2, <0.55",
+  "xclim >=0.55, <0.56",
```

Comment:
This conflicts with the `xclim` recipe in `tox.ini`. I thought we were going to maintain support for a few versions of xclim?

Reply:
Well, xclim 0.55 broke one of the missing options in `xs.extract.resample`, so that was the simple option...    I'll rewrite this PR to preserve support for >=0.53.2.

However, I think #530 needs xclim 0.55. Am I right, @coxipi  ? Are you using the fact that xclim's unit contexts can be activated for xsdba ?

Reply:
If that's the case, we can adjust it. I would simply change the `tox` entry to pin `xclim` 0.55.0 instead.

Reply:
Je confirme.